### PR TITLE
Fix typo and add QA item

### PR DESCRIPTION
**Pull request by greatbody**

### CODE REVIEW

**Comment by greatbody** [comment]
@Salvoxia Thanks for reviewing PR

File: README.md [hunk 1]
```diff
@@ -85,18 +85,18 @@ nmcli device set eth0 managed no
...
-          --bridge-eth-subnet '255.255.255.0' \
-          --bridge-eth-broadcast '192.168.0.255' \
-          --bridge-eth-gateway '192.168.0.1' \
-          --bridge-eth-mac 'b8:32:ac:8b:17:2e' \
-          --bridge-dhcp-start '192.168.0.200' \
-          --bridge-dhcp-end '192.168.0.220'
+          -u udp://VPN.SERVERNAME.COM:PORT \  # Your public IP/domain and port forwarded to Docker
+          -t \                                 # Enable TAP mode
+          -B \                                # Enable bridging mode
+          --bridge-name 'br0' \               # Bridge interface name (Optional, change if br0 already exists)
```
Comment (Salvoxia) [inline README.md]: The parameter `--bridge-name` itself is actually not optional, but changing `br0` to another name might be. `br0` is a good default value unless another bridge already exists.

**Review by Salvoxia — CHANGES_REQUESTED** [review]
Just a tiny change please, otherwise great!

File: README.md (hunk 1 above)
Comment (greatbody) [reply README.md]: THANKS, that is my misunderstanding

**Review by Salvoxia — APPROVED** [review]

**Pull request merged**